action.yml

```
@@ -46,7 +46,10 @@ runs:
 
     - name: Run Snyk to check for vulnerabilities
       shell: bash
-      run: snyk test ${{ inputs.snyk-args }} --print-deps --json --json-file-output=snyk.json
+      run: |
+        echo "::group::Snyk summary"
+        snyk test ${{ inputs.snyk-args }} --quiet --json --json-file-output=snyk.json
```

```suggestion
        snyk test ${{ inputs.snyk-args }} --print-deps --json --json-file-output=snyk.json >> snyk-debug.log
```
Imo I would not remove any flag. We just have to put all this verbose output into a file and archive it for later.